packages/next/src/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -168,6 +168,74 @@ export const processRoute = (r: Rewrite) => {
   return rewrite
 }
 
+/* Inspired by `next/src/build/webpack/react-loadable-plugin.ts` */
+const addDynamicCSSToBuildManifest = ({
+  compilation,
+  block,
+  assetMap,
+}: {
+  compilation: webpack.Compilation
+  block: webpack.AsyncDependenciesBlock
+  assetMap: DeepMutable<BuildManifest>
+}) => {
+  block.blocks.forEach((recursiveBlock) =>
+    addDynamicCSSToBuildManifest({
+      compilation,
+      block: recursiveBlock,
+      assetMap,
+    })
+  )
+
+  const dynamicCSSFiles = new Set<string>()
+
+  const chunkGroup = compilation.chunkGraph.getBlockChunkGroup(block)
+  if (chunkGroup) {
+    for (const chunk of chunkGroup.chunks) {
+      chunk.files.forEach((file: string) => {
+        if (file.endsWith('.css') && file.match(/^static\/css\//)) {
+          dynamicCSSFiles.add(file)
+        }
+      })
+    }
+  }
+
+  for (const dependency of block.dependencies) {
+    // We target the two types of dynamic import() dependencies:
+    // `ImportDependency` and `ContextElementDependency`.
+
+    // ImportDependency:
+    // - syntax: import("./module")
+    // - dependency.type: "import()"
+
+    // ContextElementDependency:
+    // - syntax: import(`./module/${param}`)
+    // - dependency.type: "import() context element"
+    if (dependency.type.startsWith('import()')) {
+      const parentModule = compilation.moduleGraph.getParentModule(dependency)
+
+      const parentChunks = compilation.chunkGraph.getModuleChunks(parentModule)
+
+      for (const chunk of parentChunks) {
+        const chunkName = chunk.name
+        if (!chunkName || !chunkName.startsWith('pages')) continue
```

Comment:
should it be startsWith `pages/`?

Reply:
`getRouteFromEntrypoint` will handle it, removed checking for `pages/`

```suggestion
        if (!chunkName) continue
```

Reply:
It seems still there, not removed